src/app/scopes-dialog/scopes.ts

```diff
@@ -789,6 +789,20 @@ export const PermissionScopes: IPermissionScope[] = [
     preview: true,
     admin: true
 },
+{
+  name: 'Whiteboards.Read',
+  description: 'Read user whiteboards',
+  longDescription: 'Allows the app to read all whiteboards you have access to, and the associated collaborators, on your behalf.',
```

Comment:
there's something a bit clumsy about the phrasing "and the associated collaborators" perhaps "their associated" would be better?

Reply:
I think we're just going to remove the " and the associated collaborators," part entirely. 
There is additional whiteboard metadata besides collaborators that you can access, so its not clear why collaborators specifically is being mentioned here.